api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -44,59 +46,82 @@
   private String username;
   // The email address that can be used to contact the user.
   private String contactEmail;
-  private String givenName;
-  private String familyName;
-  private String phoneNumber;
-  private String professionalUrl;
-  private String currentPosition;
-  private String organization;
   private Double freeTierCreditsLimitDollarsOverride = null;
-  private Short freeTierCreditsLimitDaysOverride = null;
-  private Timestamp lastFreeTierCreditsTimeCheck;
   private Timestamp firstSignInTime;
-  private Timestamp firstRegistrationCompletionTime;
   private Set<Short> authorities = new HashSet<>();
-  private Boolean idVerificationIsValid;
-  private List<Short> degrees;
-  private Timestamp demographicSurveyCompletionTime;
   private boolean disabled;
-  private Short emailVerificationStatus;
```

Comment:
an obsolete contact email check - user signup/registration takes care of this now